### PR TITLE
do not pad an empty line

### PR DESCRIPTION
**Pull request by stevemao**
Fixes the issue in https://github.com/sindresorhus/grunt-concurrent/pull/59

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,4 +1,10 @@
 'use strict';
 module.exports = function (str, pad) {
-	return pad ? pad + String(str).split(/\r?\n/).join('\n' + pad) : str;
+	return pad ? String(str).split(/\r?\n/).map(function (line) {
+		if (line) {
+			line = pad + line;
+		}
+
+		return line;
```
Comment (sindresorhus) [inline index.js]:
``` js
return line ? pad + line : line;
```

Comment (stevemao) [inline index.js]: yup done.

**Comment by kevva** [comment]
lgtm. Thanks!

**Comment by sindresorhus** [comment]
Nice catch @stevemao :)

**Comment by stevemao** [comment]
Thanks :)

**Pull request merged**